src/components/store.js

```diff
@@ -0,0 +1,7 @@
+import { createStore } from 'redux';
+import state from './components/store.js';
```

Comment:
double check this import. it is calling an error

Reply:
sorry I don't see the error coming up on my end. this is the code I have in my store.js

import state from './state';
import reducers from './reducers';
import { createStore, applyMiddleware, compose } from 'redux';
import thunk from 'redux-thunk';

const composeEnhancers =
  typeof window === 'object' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
    ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({})
    : compose;

const enhancer = composeEnhancers(applyMiddleware(thunk));
const store = createStore(reducers, state, enhancer);
export default store;